### ISSUE TITLE
Opening album/ep/single page through NewReleases returns error

### ISSUE THREAD

**Issue by TheMayhem6328** [spicetify/spicetify-cli#1501]
### 🔍 Have you checked Spicetify.app page for your issue?

- [X] I have checked the FAQ

### 🔍 Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### ℹ Environment / Computer Info

```Markdown
- Spotify version: 1.1.79.763.gc2965cdf
- Spicetify version: 2.9.1
- OS: Windows 11
```

### Upon clicking any entry in `new releases`, errors are dumped in console and I am met with a "something went wrong" page

# Below are the relevant console errors

## `vendor-xpui.js:29`
```
TypeError: Cannot read properties of null (reading 'scrollTop')
    at Grid.componentWillUnmount (spicetify-routes-new-releases.js:192:31)
    at vendor~xpui.js:29:3486748
    at Ra (vendor~xpui.js:29:3486789)
    at La (vendor~xpui.js:29:3488915)
    at Rs (vendor~xpui.js:29:3507065)
    at t.unstable_runWithPriority (vendor~xpui.js:29:3731400)
    at hi (vendor~xpui.js:29:3455825)
    at Cs (vendor~xpui.js:29:3504803)
    at gs (vendor~xpui.js:29:3493601)
    at vendor~xpui.js:29:3456097
Aa @ vendor~xpui.js:29
Ua.o.componentDidCatch.n.callback @ vendor~xpui.js:29
Vi @ vendor~xpui.js:29
ji @ vendor~xpui.js:29
Rs @ vendor~xpui.js:29
t.unstable_runWithPriority @ vendor~xpui.js:29
hi @ vendor~xpui.js:29
Cs @ vendor~xpui.js:29
gs @ vendor~xpui.js:29
(anonymous) @ vendor~xpui.js:29
t.unstable_runWithPriority @ vendor~xpui.js:29
hi @ vendor~xpui.js:29
yi @ vendor~xpui.js:29
gi @ vendor~xpui.js:29
bs @ vendor~xpui.js:29
Un @ vendor~xpui.js:29
```
## `xpui.js`
```
TypeError: Cannot read properties of null (reading 'scrollTop')
    at Grid.componentWillUnmount (spicetify-routes-new-releases.js:192:31)
    at vendor~xpui.js:29:3486748
    at Ra (vendor~xpui.js:29:3486789)
    at La (vendor~xpui.js:29:3488915)
    at Rs (vendor~xpui.js:29:3507065)
    at t.unstable_runWithPriority (vendor~xpui.js:29:3731400)
    at hi (vendor~xpui.js:29:3455825)
    at Cs (vendor~xpui.js:29:3504803)
    at gs (vendor~xpui.js:29:3493601)
    at vendor~xpui.js:29:3456097
a @ xpui.js:1
onError @ xpui.js:1
componentDidCatch @ xpui.js:1
Ua.o.componentDidCatch.n.callback @ vendor~xpui.js:29
Vi @ vendor~xpui.js:29
ji @ vendor~xpui.js:29
Rs @ vendor~xpui.js:29
t.unstable_runWithPriority @ vendor~xpui.js:29
hi @ vendor~xpui.js:29
Cs @ vendor~xpui.js:29
gs @ vendor~xpui.js:29
(anonymous) @ vendor~xpui.js:29
t.unstable_runWithPriority @ vendor~xpui.js:29
hi @ vendor~xpui.js:29
yi @ vendor~xpui.js:29
gi @ vendor~xpui.js:29
bs @ vendor~xpui.js:29
Un @ vendor~xpui.js:29
```

### 📸 Screenshots

![image](https://user-images.githubusercontent.com/56154436/155642382-015c43b9-3bcd-48ff-989a-e719d32cfac2.png)
![image](https://user-images.githubusercontent.com/56154436/155642272-18e8a615-9296-4466-b214-12097e355f83.png)

**Comment by afonsojramos** [comment]
I can confirm that this is happening. Will look into it tomorrow, but it should be an easy fix.

**Comment by afonsojramos** [comment]
Closed by #1510 and #1511.
